interactive_textinput_printer.go

```diff
@@ -101,7 +101,7 @@ func (p InteractiveTextInputPrinter) Show(text ...string) (string, error) {
 	area.StartOfLine()
 
 	if !p.MultiLine {
-		cursor.Right(len(RemoveColorFromString(areaText)))
+		cursor.Right(len([]rune(RemoveColorFromString(areaText))))
```

Comment:
```suggestion
		cursor.Right(runewidth.StringWidth((RemoveColorFromString(areaText))))
```

Hi, we use the `StringWidth` method in other places too, so we would prefer to use it here too. 
Could you try if that works for your characters, and if so, update the PR? Thanks!

Reply:
Hi, this works perfectly, thanks!